drivers/sensor/hmc5883l/Kconfig

```diff
@@ -20,7 +20,7 @@ config HMC5883L_NAME
 config HMC5883L_I2C_MASTER_DEV_NAME
 	string "I2C master where HMC5883L is connected"
 	depends on HMC5883L
-	default "I2C_0"
+	default "I2C_1"
```

Comment:
why are you changing this here? If you need that for you sample, you can set to I2C_1 in your sample.

Reply:
Reverted to original code and declared in prj.conf.